packages/ethereum/web3-ethereum/src/index.ts

```diff
@@ -127,14 +113,8 @@ export class Web3Ethereum implements EthereumProvider.Ethereum {
 
 	async getBalance(address: Address): Promise<BigNumber> {
```

Comment:
Why this function is relying on BigNumber of rarible/types
it's a string big number that related to our backend-generated api clients 

here your getBalance should return BigNumber of rarible/utils since it's bignumber impl for our internal usage across all of our packages